packages/components/src/item-group/styles.ts

```diff
@@ -23,6 +23,16 @@ export const unstyledButton = css`
 	&:hover {
 		color: ${ COLORS.ui.theme };
 	}
+
+	&:focus {
+		box-shadow: 0 0 0 var( --wp-admin-border-width-focus )
```

Comment:
I believe the `box-shadow` used [on the inserter](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/inserter-list-item/style.scss#L84) has `inset`.

Reply:
I see the inserter uses `inset`, but I was trying to align to the styles used in the block settings panel, and these seem to use [these styles](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/sidebar-button/style.scss#L11) which do not use `inset`. I think the biggest difference with using `inset` is that it removes the border radius:

| Without Inset | With Inset |
| ------------- | ---------- |
| <img width="349" alt="image" src="https://github.com/WordPress/gutenberg/assets/1645628/5a551e82-acec-4aa4-a41a-06a134b44319"> | <img width="349" alt="image" src="https://github.com/WordPress/gutenberg/assets/1645628/edb6f07f-71e2-45d2-8f21-b888d2740c98"> |

In the [original issue](https://github.com/WordPress/gutenberg/issues/50757), it seems that the version with a border radius is preferable, but perhaps the inserter should also be updated to match these styles too.

Reply:
Thank you, Sarah! That makes sense; I was considering the inserter as the reference.

Reply:
No worries, I also first looked at the inserter styles and then noticed they were slightly different from the screenshots in the original issue. We could follow up with another PR to make the inserter buttons use the same focus style as well.